app/src/main/java/com/willowtree/vocable/facetracking/FaceTrackingViewModel.kt

```diff
@@ -78,6 +82,10 @@ class FaceTrackingViewModel : ViewModel(), KoinComponent {
                         liveAdjustedVector.postValue(oldVector)
                     }
                     else -> {
+                        val isTablet = get<Context>().resources.getBoolean(R.bool.is_tablet)
+                        if (!isTablet) {
+                            y *= 2f
```

Comment:
will this keep multiplying y by 2 every time we hit this method? So will y keep growing and growing?

Reply:
I don't think so, also you should pull this down and test it on your own device if u can!